docs/guides/build-xrd.md

```diff
@@ -0,0 +1,202 @@
+{{< /hint >}}
+
+{{< img src="/docs/media/basic-compositions/ref-configs.png" alt="Marketplace reference Configurations" size="small" >}}
+
+Click into the `platform-ref-gcp` Configuration  and click the link on the next page under “Source Code” to go to the source repo.
+
+{{< img src="/docs/media/basic-compositions/config-source.png" alt="Reference Configuration source" size="small" >}}
+
+Within this GitHub repo, you are going to copy an existing XRD’s definition. You are going to use the PostgreSQL instance XRD and repurpose it for your needs. Copy the contents of the Postgre SQL [XRD definition file](https://raw.githubusercontent.com/upbound/platform-ref-gcp/main/package/database/postgres/definition.yaml) and paste it into your `definition.yaml` created earlier.
+
+{{< hint "tip" >}}
+An aside: these Configurations in the Marketplace are reference implementations. In some cases, you may want to build an API for which there is already an implementation (like a Kubernetes cluster or PostgreSQL instance). It is completely fair practice to use these as launching pads, but unfortunately for our VM scenario, it doesn’t exist and so we have to build it from scratch. Nevertheless, we can copy the .yaml and use some of it as boilerplate code.
+{{< /hint >}}
+
+## Build the XRD
+
+Next, prune out all the fields that are irrelevant for your use case (i.e. they are specific to PostgreSQL) and you will end up with this. This is boilerplate.yaml that nearly all XRDs will want to have:
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+ name: <todo>
+spec:
+ group: <todo>
+ names:
+   kind: <todo>
+   plural: <todo>
+ claimNames:
+   kind: <todo>
+   plural: <todo>
+ versions:
+ - name: v1alpha1
+   served: true
+   referenceable: true
+   schema:
+     openAPIV3Schema:
+       type: object
+       properties:
+         spec:
+           type: object
+           properties:
+             parameters:
+               type: object
+               properties:
+               required:
+           required:
+             - parameters
+```
+
+{{< hint "tip" >}}
+The Composition [reference docs]({{<ref "master/reference/composition.md" >}}) are an invaluable reference at this time.
+{{< /hint >}} 
+
+Going by the conventions as described in the Composition reference docs, you will fill in values for each of the `todo`. Fill in the top half of the config as below:
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: xvminstances.compute.acme.co
+spec:
+  group: compute.acme.co
+  names:
+    kind: XVMInstance
+    plural: xvminstances
+  claimNames:
+    kind: VMInstance
+    plural: vminstances
+...
+```
+
+Now to explain what you just wrote:
+
+* The `group` is the declaration of where this API will live (in the future you could define multiple APIs and have them all be a member of the ‘compute.acme.co’ group).

```

Comment:
Optional: you may want to consider using the new `hover highlight` docs feature. It's designed for exactly this use case of calling out lines of larger outputs. 

https://docs.crossplane.io/master/contributing/docs/#syntax-highlighting